frontend/css/brainzplayer.less

```
@@ -351,6 +351,23 @@
   }
 }
 
+.volume {
+  position: fixed;
+  width: fit-content;
+  z-index: 109;
+  height: 10em;
+  background-color: #f8f8f8;
+  right: 16em;
+  .volume-input {
+    width: 7em;
+    margin-top: @brainzplayer-height;
+    transform: rotate(270deg);
```

I have a fun CSS trick for you today!
For vertical range inputs, you can use:
```suggestion
    writing-mode: vertical-rl;
    direction: rtl;
    margin: 1.5em 0.6em;
```